etc/waffle/nginx/vhost.d/poxa.conf

```diff
@@ -0,0 +1,13 @@
+location /app/ {
```

Comment:
We need the vhost config, but it should definitely not be hardcoded to my hostname, ha. Let me make that a dynamic.